src/routes/InstancesPage/DeleteInstanceModal.tsx

```diff
@@ -9,9 +9,17 @@ import {
   ModalVariant,
   TextInput,
 } from '@patternfly/react-core';
-import React, { useState } from 'react';
+import React, { ReactElement, useState } from 'react';
+import { Instance } from '../../hooks/apis/useInstances';
 
-function DeleteInstanceModal({ isOpen, instance, onRequestDelete, onClose }) {
+type DeleteInstanceModalProps = {
+  isOpen: boolean;
+  instance: Instance; // @TODO: make actual type
```

Comment:
Is `Instance` just a placeholder pattern for "instances" of some entity in the future, when we move away from the dinosaur API? Or is it closer to an instance of a managed ACS install?

Reply:
Yea, eventually we'll need to use something more appropriate. Seems like the naming for the resources is going to be something like `/api/rhacs/v1/centrals/{id}` (https://issues.redhat.com/browse/ROX-9789). Maybe instead of `Instance` we can agree to use `Central`